lib/stacks/api/appConfiguration.ts

```diff
@@ -68,6 +68,7 @@ export class AppConfigurationApiStack extends Stack {
                 restApi,
                 props.authorizer,
                 system_permissions.includes(f.name) ? props.systemRole : props.applicationRole,
```

Comment:
Same thing here - we sometimes use the system role, is that going to create a problem?